translate/pb/pb_decoder_entry.go

```diff
@@ -10,46 +10,29 @@ import (
 	"google.golang.org/protobuf/types/dynamicpb"
 )
 
-// StdProtobufDecoder transforms the protocol buffer message to a tuple.
```

Comment:
Nit: both of these vars should have at least a one-line comment describing what they do.